.github/workflows/docs/build-docs

```diff
@@ -59,16 +71,24 @@ def build_module_docs(module):
         content = f.readlines()
         content.append("\n.. toctree::\n\t:caption: Extensions:\n\t:maxdepth: 1\n\n")
         content.append(f"\tIntroduction <../index.html#http://>\n")
-        for mod in get_all_modules():
```

Comment:
I guess `get_all_modules()` and maybe some other functions can be deleted?

Reply:
I have done some clean up work in that file, removed this and other things.